patches/src/main/kotlin/app/revanced/patches/cricbuzz/DisableAdsPatch.kt

```diff
@@ -0,0 +1,33 @@
+package app.revanced.patches.cricbuzz
+
+import app.revanced.patcher.extensions.InstructionExtensions.addInstructions
+import app.revanced.patcher.extensions.InstructionExtensions.getInstruction
+import app.revanced.patcher.fingerprint
+import app.revanced.patcher.patch.bytecodePatch
+import com.android.tools.smali.dexlib2.Opcode
+import com.android.tools.smali.dexlib2.iface.instruction.formats.Instruction11x
+
+@Suppress("unused")
+val disableAdsPatch = bytecodePatch (
+    "Disable ads",
+) {
+    compatibleWith("com.cricbuzz.android")
+
+    execute {
+        // modify user state as "ACTIVE" during switch statement
+        userStateSwitchFingerprint.method.apply {
+            // find the line where the switch variable is assigned
+            // and replace the register with a constant string "ACTIVE"
```

Comment:
Code is self explanatory already
```suggestion
        userStateSwitchFingerprint.method.apply {
```